panda_gym/envs/panda_tasks.py

```diff
@@ -22,27 +21,46 @@ class PandaFlipEnv(RobotTaskEnv):
         reward_type (str, optional): "sparse" or "dense". Defaults to "sparse".
         control_type (str, optional): "ee" to control end-effector position or "joints" to control joint values.
             Defaults to "ee".
-        render (bool, optional): Deprecated: This argument is deprecated and will be removed in a future
-            version. Use the render_mode argument instead.
+        renderer (str, optional): Renderer, either "Tiny" or OpenGL". Default to "Tiny".
```

Comment:
Missing s: Default**s**

Also, maybe mention that only OpenGL is available for `render_mode='human'`.

Applies to all tasks here.